Orlando Walking Tours/Controllers/LocationListVC.swift

```diff
@@ -254,6 +285,18 @@ class LocationListVC: UIViewController
 
     ////////////////////////////////////////////////////////////
 
+    func sortAlphabetically()
+    {
+        self.locations.sort
+        {
+            let a = $0
+            let b = $1
+            return a.locationTitle?.localizedCaseInsensitiveCompare(b.locationTitle!) == ComparisonResult.orderedAscending
```

Comment:
never force unwrap unless the initialisation is directly above in the same scope.